alembic/versions/28747295af93_add_filter_status_field_to_product_model.py

```diff
@@ -13,7 +13,7 @@
 
 # revision identifiers, used by Alembic.
 revision: str = '28747295af93'
-down_revision: Union[str, None] = '5f9d01dc6cc9'
+down_revision: Union[str, None] = '5f8e13d19ccb'
 branch_labels: Union[str, Sequence[str], None] = None
```

Comment:
you updated migrations?
revert this please

Reply:
Okay now look at what happens when I do

![image](https://github.com/user-attachments/assets/23df39c3-675c-4a85-8a32-83bac04a4d56)

Reply:
![image](https://github.com/user-attachments/assets/9545f984-fb1e-4dac-8d7b-9898daaab1fd)